module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,19 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+bool compare(const std::string& s1, const std::string& s2) {
+    return s1.size() < s2.size();
+}
+
+std::deque<std::string> lengthSort(std::forward_list<std::string>& str) {
+    str.sort();
+    std::deque<std::string> sortMe;
+
+    for (const auto& el : str) {
+        sortMe.push_back(el);
+    }
```

Comment:
it's much better to initialize sortMe with the values from the forward_list immediately on creation with a proper constructor usage. Pushing_back is not efficient.

BTW str as a name of forward_list is misleading.